app/commands/v2/import.rb

```diff
@@ -0,0 +1,98 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
+        payload[:content_items].map.with_index do |event, index|
+          create_content_item(event, index, payload[:content_id])
+        end
+
+        after_transaction_commit do
+          send_downstream(payload[:content_items].find { |e| e[:action] == 'Publish' })
```

Comment:
From my understanding this will only update the live content store? and that is only when the `:action` is `Publish`, this seems to be the only action that will update the content store so no items will update the draft content store? and this usage seems somewhat confusing if the final state is unpublished


also quibble single quotes